Shared/Samples/Set feature layer rendering mode on scene/SetFeatureLayerRenderingModeOnSceneView.swift

```diff
@@ -0,0 +1,147 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SetFeatureLayerRenderingModeOnSceneView: View {
+    /// Scene that displays with dynamic rendering.
+    @State private var dynamicScene: ArcGIS.Scene = {
+        let scene = Scene()
+        scene.initialViewpoint = .zoomedOut
+        return scene
+    }()
+    
+    /// Scene that displays with static rendering.
+    @State private var staticScene: ArcGIS.Scene = {
+        let scene = Scene()
+        scene.initialViewpoint = .zoomedOut
+        return scene
+    }()
+    
+    /// A Boolean value indicating whether the scene views are currently zooming.
+    @State private var isZooming = false
+    
+    /// The viewpoint for the scene.
+    @State private var viewpoint: Viewpoint?
+    
+    /// A Boolean value indicating whether the scene is fully zoomed in.
+    @State private var isZoomedIn = true
+    
+    /// Creates service feature tables using point, polygon, and polyline services
+    let featureTables: [ServiceFeatureTable] = [
+        ServiceFeatureTable(url: .pointTable),
+        ServiceFeatureTable(url: .polylineTable),
+        ServiceFeatureTable(url: .polygonTable)
+    ]
+    
+    init() {
+        /// Iterate through the feature tables and use them to setup a feature layers. Set the rendering mode
+        /// for either dynamic or static rendering and add the feature layer to the scene.
```

Comment:
A few fixes I found here:
- "a feature layers" should be "feature layers".
- Use "set up" instead of "setup".
- Multiple feature layers are added, so switch to the plural form.

```suggestion
        /// Iterate through the feature tables and use them to set up feature layers. Set the rendering mode
        /// for either dynamic or static rendering, and add the feature layers to the scene.
```

Finally, since this is an inline and not a doc comment, use `//` instead of `///` for these lines:

```suggestion
        // Iterate through the feature tables and use them to set up feature layers.
        // Set the rendering mode for either dynamic or static rendering,
        // and add the feature layers to the scene.
```